docs/glossary.md

```diff
@@ -179,6 +179,14 @@ An append-only log of Events that represents the full state a Workflow Execution
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
+<span style={{ backgroundColor: 'rgb(255, 255, 102)', fontWeight: 'bold' }}>When MRN goes live update this link from self-hosted-guide/multi-cluster-replication to cloud/multi-region</span>
+#### [Failover](/self-hosted-guide/multi-cluster-replication) <!--[Failover](/cloud/multi-region)-->
+
+A failover shifts Workflow Execution processing from an active Cluster to a standby [Cluster](#temporal-cluster) during outages or other incidents.
+Standby Clusters use [Multi-Cluster Replication](#multi-cluster-replication) to duplicate data and prevent data loss during failover.
```

Comment:
Let's either drop this entirely, or rewrite it to mention MRN for Cloud and Multi-Cluster Replication for Self-hosting.

Reply:
replacing MCR with replication and no link